packages/dds/map/src/map.ts

```diff
@@ -245,14 +247,30 @@ export class SharedMap extends SharedObject<ISharedMapEvents> implements IShared
 		this.kernel.clear();
 	}
 
+	/**
+	 * Get the attribution of one entry through its key
+	 * @param key - Key to track
+	 * @returns The attribution of related entry
+	 * @alpha
+	 */
+	public getAttribution(key: string): AttributionKey | undefined {
+		return this.kernel.getAttribution(key);
+	}
+
+	/**
+	 * Get all attribution of the map
+	 * @returns All attribution in the map
+	 * @alpha
+	 */
+	public getAllAttribution(): Map<string, AttributionKey> | undefined {
+		return this.kernel.getAllAttribution();
+	}
+
 	/**
 	 * {@inheritDoc @fluidframework/shared-object-base#SharedObject.summarizeCore}
 	 * @internal
 	 */
-	protected summarizeCore(
-		serializer: IFluidSerializer,
-		telemetryContext?: ITelemetryContext,
-	): ISummaryTreeWithStats {
+	protected summarizeCore(serializer: IFluidSerializer): ISummaryTreeWithStats {
```

Comment:
i'd revert the signature change to this method. no reason to modify that in this PR